docs: add HDD-supported document (#213)
harvester/docs: docs/advanced/storageclass.md

1. `Immediate`: Binds and provisions a persistent volume once the PersistentVolumeClaim is created.
2. `WaitForFirstConsumer`: Binds and provisions a persistent volume once a VM using the PersistentVolumeClaim is created.

![](/img/v1.1/storageclass/customize_tab_vol_binding_mode.png)

> # Appendix - Use Case
> 
> ## HDD scenario

Reviewer: ```suggestion
## Appendix - Use Case

### HDD Scenario
```

Reviewer: Please do not use more than one level-1 headings in one single document, because level-1 headings will not show in the right sidebar

Author: OK, thanks for the tips!

> After introducing *StorageClass*, Harvester now could use **HDD** for the cold storage of tiering or archiving.

Reviewer 2: ```suggestion
With the introduction of *StorageClass*, users can now use HDDs for tiered or archive cold storage.
```

Author: done

> After introducing *StorageClass*, Harvester now could use **HDD** for the cold storage of tiering or archiving.

Reviewer: ```suggestion
After introducing *StorageClass*, Harvester can now use **HDD** for cold storage tiering and archiving.
```

Author: done, thanks!

:::caution

> HDD is not recommended for guest RKE2 clusters or regular use.

Reviewer 2: ```suggestion
HDD is not recommended for guest RKE2 clusters or VMs with good performance disk requirements.
```

Author: done

> HDD is not recommended for RKE2 clusters and regular use.

Reviewer: ```suggestion
HDD is not recommended for RKE2 clusters or regular use.
```

Author: done

:::

> #### Recommand Partice

Reviewer 2: ```suggestion
#### Recommended Practice
```

Author: done

> ### Recommand Partice

Reviewer: ```suggestion
#### Recommended Practices
```

Author: done

> First, add your HDD on the `Host` page and specify the disk tags like `HDD` or `ColdStorage` as needed. For more information about how to add extra disks and disk tags, see [Multi-disk Management](https://docs.harvesterhci.io/v1.1/host/#multi-disk-management).

Reviewer 2: ```suggestion
First, add your HDD on the `Host` page and specify the disk tags as needed, such as`HDD` or `ColdStorage`. For more information on how to add extra disks and disk tags, see [Multi-disk Management](https://docs.harvesterhci.io/v1.1/host/#multi-disk-management) for details.
```

Author: done

![](/img/v1.1/storageclass/add_hdd_on_host_page.png)

![](/img/v1.1/storageclass/add_tags.png)

> Then, create a new `StorageClass` for the HDD (with the disk tag above). For HDDs that offer large capability but slow performance, you can reduce the number of replicas to improve the performance. For details, see [storageclass](https://docs.harvesterhci.io/v1.1/advanced/storageclass).

Reviewer 2: ```suggestion
Then, create a new `StorageClass` for the HDD (use the above disk tags). For hard drives with large capacity but slow performance, the number of replicas can be reduced to improve performance. For details, see [storageclass](https://docs.harvesterhci.io/v1.1/advanced/storageclass) for details.
```

Author: done

![](/img/v1.1/storageclass/create_hdd_storageclass.png)

> Now, you can create a volume with the above `StorageClass` used by the HDD for cold storage or archive.

Reviewer 2: ```suggestion
You can now create a volume using the above `StorageClass` with HDDs mostly for cold storage or archiving.
```

Author: done
